### PR TITLE
Feature/fatma python 1

### PR DESCRIPTION
**Pull request by fatmaQunnies**
In this task, I created a list of books and added details such as title, author, availability, and tags for each book. Then, I found the books by authors named "Mohammed" that are available. Finally, I checked the first tag of each book and printed a message about the book based on that tag.

### CODE REVIEW

**Comment by Mohammed-samaheen** [comment]
you should not remove the other files and folders in the repo, only add your file

**Comment by Mohammed-samaheen** [comment]
If you are using Windows and faced an issue when cloning the repo, I Updated the <first_name>_<last_name>.py file name to fix that issue.

File: README.md
Comment (Mohammed-samaheen) [inline README.md:1]: you removed the readme file

File: Fatma_Qunnies.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        available_books_by_mohammed.append(book["title"])
+        print(book["availability"])
+
+# print 
+print(available_books_by_mohammed)
+
+
+# Step 3: Analyze Tags with Match Statement
+
+def matchs(book):
```
Comment (Mohammed-samaheen) [inline Fatma_Qunnies.py:32]: It's good to use a function, but it's not within the scope of this task.

File: Fatma_Qunnies.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    },{
+        "title":"Data Structures in Python",
+        "author":"Yousef Amr",
+        "availability": True,
+        "tags": ["Data","Python","Intermediate"]}]
+# Step 2: Filter Books by Author
+available_books_by_mohammed = []
+for book in books:
+    if "Mohammed" in book["author"] :
+        book["availability"] = True
```
Comment (Mohammed-samaheen) [inline Fatma_Qunnies.py:22]: Double-check this part; it's not true.

**Comment by fatmaQunnies** [comment]
In this task, I created a list of books and added details such as title, author, availability, and tags for each book. Then, I found the books by authors named "Mohammed" that are available. Finally, I checked the first tag of each book and printed a message about the book based on that tag.

**Comment by Mohammed-samaheen** [comment]
> In this task, I created a list of books and added details such as title, author, availability, and tags for each book. Then, I found the books by authors named "Mohammed" that are available. Finally, I checked the first tag of each book and printed a message about the book based on that tag.

 description should be here

image

File: Fatma_Qunnies.py (hunk 1 above)
Comment (Mohammed-samaheen) [reply Fatma_Qunnies.py:31]: > It's good to use a function, but it's not within the scope of this task.

you still need to fix this

**Review by FaresAbuIram — APPROVED** [review]
LGTM, Good job

**Pull request merged**